glide.lock

```diff
@@ -1,59 +1,91 @@
-hash: 984ca177636c530f9ba81275484561386daf42454e58d37c8b598b1e49631a70
-updated: 2016-11-17T17:59:46.025091418+01:00
+hash: 4b7e766a6892b1292498602d53373b22148f11b9688ee1f5f2390b182d600655
+updated: 2017-08-04T11:23:09.070809824-07:00
 imports:
-- name: github.com/intelsdi-x/snap
-  version: 4b6dc4090ac3ab0f800e656bfe3584e30f25c12d
-  subpackages:
-  - control/plugin
-  - control/plugin/cpolicy
-  - control/plugin/encoding
-  - control/plugin/encrypter
-  - core
-  - core/cdata
-  - core/ctypes
-  - core/serror
-  - pkg/ctree
-  - pkg/schedule
-  - pkg/stringutils
-  - scheduler/wmap
-- name: github.com/robfig/cron
-  version: 32d9c273155a0506d27cf73dd1246e86a470997e
+- name: github.com/golang/protobuf
+  version: 888eb0692c857ec880338addf316bd662d5e630e
+  subpackages:
+  - proto
+  - ptypes/any
+- name: github.com/intelsdi-x/snap-plugin-lib-go
+  version: a8b9252d1c8305deb3c24495f1e9ce26607fcbd7
```

Comment:
@rashmigottipati, could you point to the latest snap-plugin-lib-go (4fa1360e35b2f1ebb5fb14a575cc11ecc199c531)?
Please, also check if after this update, diagnostic runs fine without providing config (defaults should be applied)